packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3015,30 +3015,30 @@ void main() {
       // Cursor move doesn't trigger a toolbar initially.
       expect(find.byType(CupertinoButton), findsNothing);
 
-      await gesture.moveBy(const Offset(50, 0));
+      await gesture.moveBy(const Offset(100, 0));
```

Comment:
why this change?

Reply:
So this test was failing because it attempts to drag the selection onto the white space following a word.

Take 'Atwater Peel Sherbrooke Bonaventure', if we select 'Atwater' and try to drag the selection over the white space following it, then the selection will skip the whitespace and drag onto the next word. I checked this behavior on uneditable text on iOS and it looks like it is consistent. Though I'm not sure if this behavior is what is desired.

Reply:
Indeed on master I see that I cannot drag the selection handle after selecting a word to select the space after it, on both a TextField and a SelectableText.  It jumps into the next word.  Maybe we're not detecting the drag gesture soon enough?

On native iOS I am able to select the space, so this seems like something that should be fixed (but not related to this PR).

I don't see how this test was able to pass in the first place, but if it now mimics a test in CupertinoTextField then I guess it's ok for now?

Reply:
Yeah i'm not sure how it still passed. I also checked the behavior on master and came to the same conclusion you did.